lib/os/Kconfig

```diff
@@ -83,11 +83,13 @@ choice SPSC_PBUF_CACHE_HANDLING
 
 config SPSC_PBUF_CACHE_FLAG
 	bool "Use cache flag"
+	imply CACHE_MANAGEMENT
```

Comment:
Shouldn't this be a "depends on"?  Why would someone demand the use of the cache API in spsc_pbuf if they didn't want it to work?